lib/3scale/core/users.rb

```diff
@@ -0,0 +1,24 @@
+module ThreeScale
+  module Core
+    class Users < APIClient::Resource
```

Comment:
Shouldn't this be in the existing `User` class?

Reply:
I think that having a single class is more common. You could give the method a different name to be more explicit like `User.delete_all_for_service(service_id)`.

Reply:
@unleashed your call

Reply:
I also think this belongs in `User`. Having a class for this with the plural form is confusing to me (I also might be spoiled by Rails model conventions).